src/index.js

```diff
@@ -1,100 +1,138 @@
-import { initializeService } from './services/config.js';
+/*** This file was generated automatically. To recreate, please run `npm run build-index`. ***/
 
 import {
-  fetchSongById,
-  fetchArtists,
-  fetchSongArtistCount,
-  fetchRelatedSongs,
-  fetchAllSongs,
-  fetchSongFilterOptions,
-  fetchSongCount,
-  fetchWorkouts,
-  fetchNewReleases,
-  fetchUpcomingEvents,
-  fetchByRailContentId,
-  fetchByRailContentIds,
-  fetchAll,
-  fetchAllFilterOptions,
-  fetchMethodNextLesson,
-  fetchFoundation,
-  fetchMethod,
-  fetchMethods,
-  fetchMethodChildren,
-  fetchMethodChildrenIds,
-  fetchNextPreviousLesson,
-  fetchRelatedLessons,
-  fetchAllPacks,
-  fetchPackAll,
-  fetchPackChildren,
-  fetchLessonContent,
-  fetchCourseOverview,
-  fetchChildren,
-  fetchParentByRailContentId,
-  fetchMethodPreviousNextLesson,
-  fetchLiveEvent,
-  fetchChallengeOverview,
-  fetchCoachLessons,
-  fetchByReference,
-  fetchScheduledReleases,
-  fetchArtistLessons,
-  fetchGenreLessons,
-  fetchRelatedMethodLessons,
-} from './services/sanity.js';
+	globalConfig,
+	initializeService
+} from './services/config.js';
 
-import { 
-  fetchCompletedState, 
-  fetchAllCompletedStates,
-  fetchContentInProgress,
-  fetchCompletedContent,
-  fetchVimeoData,
-  fetchContentPageUserData,
+import {
+	fetchAllCompletedStates,
+	fetchCompletedContent,
+	fetchCompletedState,
+	fetchContentInProgress,
+	fetchContentPageUserData,
+	fetchHandler,
+	fetchLikeContent,
+	fetchSongsInProgress,
+	fetchUnlikeContent,
+	fetchUserContext,
+	fetchVimeoData
 } from './services/railcontent.js';
```

Comment:
This adds entries that are exported in railcontent.js but were missing before:

- fetchHandler,
- fetchLikeContent,
- fetchSongsInProgress,
- fetchUnlikeContent,
- fetchUserContext,